Assets/Scripts/BehaviorTree

```diff
@@ -1 +1 @@
-Subproject commit ff882fc9ddda542b10c40b3c31025fd7dee2e8f4
+Subproject commit 103f81c73540e85d02dbb91570cca33230d68ebd
```

Comment:
This is the way that git tracks submodules, it puts a file where the submodule would be that specifies the commit in the submodule that you want to be on. If you see Behavior Tree under unstaged files you can go ahead and add it to your next commit. That being said, it shouldn't be showing up as changed so often, and it's been causing some problems. I'll try to figure out what can be done to make this less of any issue, I might just remove the submodule and return it to being a normal file.